### PR TITLE
update README to reflect v1

### PR DESCRIPTION
**Pull request by calvinnhieu**
+ supporting documentation https://allspark.dev.data.humancellatlas.org/HumanCellAtlas/matrix-service/wikis/Technical-Architecture

### CODE REVIEW

**Comment by brianraymor** [comment]
For #322 _The difference in the v0 and v1 endpoints don't seem to be explained._ - does the README clearly differentiate v0 (contract with the data browser - not intended for humans) and v1 (developer friendly)?

File: README.md [hunk 1]
```diff
@@ -1,79 +1,135 @@
...
-The API also makes this information available via the `/matrix/formats` endpoint.
+This list is also available at `/v1/formats` with additional information for a specific format available at
+`/v1/formats/<format>`.
+
+### Feature
 
-## Getting Started
+The Matrix Service also supports generating cell by transcript matrices in addition to cell by gene matrices. To select
+the feature type, specify either `gene` (default) or `transcript` in the POST request. The list of available features is
+available at `/v1/features` with additional information for a specific feature available at `/v1/features/<feature>`.
```
Comment (mckinsel) [inline README.md]: "Note that some assay types are incompatible with certain feature types. For example, if `transcript` is selected as the feature type, data from 3' assays will not be included in the generated matrix."

File: README.md [hunk 2]
```diff
@@ -1,79 +1,135 @@
...
-### File formats
+### Format
 
-The DCP MS enables users to prepare expression matrices in several formats by supplying the `format` parameter in the
-POST request to the `/matrix` endpoint. The following is a list of supported file formats:
+The Matrix Service supports generating matrices in the following 3 formats:
 
-- [.zarr](https://zarr.readthedocs.io/en/stable/) (default)
-- [.loom](http://loompy.org/)
+- [.loom](http://loompy.org/) (default)
```
Comment (mckinsel) [inline README.md]: Should we remove the leading `.`s here? I wonder if people will think you're supposed to include them in the request body.

File: README.md [hunk 3]
```diff
@@ -1,79 +1,135 @@
...
-|---|---|
-| Cache table | Caches requests by a hash of its input parameters. |
-| State table | Tracks the progress of a request. |
-| Output table | Stores output values of the request (e.g. file format, errors). |
-| Lock table | Manages locks for across distributed nodes. |
+### Fields
 
+Users can specify a list of metadata fields to be exported with an expression matrix. The list of available metadata
+fields is available at `/v1/fields`. More information about a specific field is available at
+`/v1/fields/<field>`.
```
Comment (mckinsel) [inline README.md]: "By default, all available metadata fields are included in the generated matrix."

File: README.md [hunk 4]
```diff
@@ -1,79 +1,129 @@
 # Expression Matrix Service
 
 [![Production Health Check](https://status.data.humancellatlas.org/service/matrix-health-check-prod.svg)](https://matrix.data.humancellatlas.org/)
-[![Master Deployment Status](https://status.data.humancellatlas.org/build/HumanCellAtlas/matrix-service/prod.svg)](https://allspark.dev.data.humancellatlas.org/HumanCellAtlas/matrix-service/pipelines)
```
[the remark this answers is not in the document]
Comment (calvinnhieu) [reply README.md]: Hm I think the badge we were using was hosted on @mweiden's service status dashboard which made them public. Not sure what happened, but those badges don't work anymore. GitLab badges still work, but you have to authenticate to see them which doesn't work in a README:

https://allspark-prod.data.humancellatlas.org/HumanCellAtlas/matrix-service/badges/prod/pipeline.svg

**Comment by calvinnhieu** [comment]
@brianraymor addressed, in Versions section

**Pull request merged**